docs/guides/dkms.md

```diff
@@ -125,13 +125,17 @@ If you still face an issue, mention it [here](https://github.com/t2linux/wiki/is
 
 Copy [this script](../tools/rmmod_tb.sh) to `/lib/systemd/system-sleep/rmmod_tb.sh`
 
+If you are using Gentoo with openrc, instead copy the script to `/lib64/elogind/system-sleep/rmmod_tb.sh`
+
 Now run :-
 
 ```sh
 sudo chmod 755 /lib/systemd/system-sleep/rmmod_tb.sh
 sudo chown root:root /lib/systemd/system-sleep/rmmod_tb.sh
 ```
 
+Change the path to `/lib64/elogind/system-sleep/rmmod_tb.sh` if using openrc on Gentoo as mentioned previously.
```

Comment:
Isn't this implied from the previous line you added?

Reply:
I guess what @vi-improved did is good enough in this case.